client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1468,7 +1468,9 @@ def _get_representations(self, placeholder):
         product_name_regex = None
         if product_name_regex_value:
             product_name_regex = re.compile(product_name_regex_value)
-        product_type = placeholder.data["family"]
+        product_type = placeholder.data.get("product_type")
+        if product_type is None:
+            product_type = placeholder.data.get("family")
```

Comment:
Should we allow both of these to ever not exist without error?

Reply:
Nope
```suggestion
            product_type = placeholder.data["family"]
```